Dockerfile

```diff
@@ -10,4 +10,7 @@ WORKDIR /app
 
 COPY --from=builder /code/build/target/processor /app/run
 
+# Let Go know how much memory it has available.
+ENV GOMEMLIMIT="400MiB"
+
```

Comment:
Go potrebuje vediet kolko memory ma k dispozcii.
- By default to berie z `ulimit`, ktory ale nase komponenty nemaju nastavene.
- V starsom Go to neslo inak nastavit, ale uz tam je nejaky cas tato ENV.
  - Viac info: https://weaviate.io/blog/gomemlimit-a-game-changer-for-high-memory-applications